ooni/resources.py

```diff
@@ -0,0 +1,184 @@
+import json
+import errno
+
+from twisted.python.filepath import FilePath
+from twisted.internet import defer
+from twisted.web.client import downloadPage, getPage, HTTPClientFactory
+
+# WARNING: this script is being run as part of the post install procedure.
+# Be sure to not import either in this module or in the imported modules
+# dependencies other than twisted. If you end up including something that is
+# not twisted, then you will need to add it to the setup_requires in setup.py.
```

Comment:
This comment should actually be deleted since it's no longer accurate. The setup.py doesn't import the resources anymore and doesn't run it as part of the postinstall.